packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -298,6 +298,27 @@ document.getElementById('dialer').addEventListener('submit', (event) => {
   // again, we don't want to reload when we try to dial
   event.preventDefault();
   return spark.meetings.create(document.getElementById('invitee').value).then((m) => {
+    console.error(m);
+    startMeeting(m);
+  });
+});
+
+// And finally, let's wire up dialing
+document.getElementById('conversation-dialer').addEventListener('submit', (event) => {
+  // again, we don't want to reload when we try to dial
+  event.preventDefault();
+  return spark.meetings.create(document.getElementById('invitee-convo').value).then((m) => {
+    console.error(m);
+    startMeeting(m);
+  });
+});
+
+// And finally, let's wire up dialing
+document.getElementById('type-dialer').addEventListener('submit', (event) => {
+  // again, we don't want to reload when we try to dial
+  event.preventDefault();
+  return spark.meetings.createWithType(document.getElementById('type').value, document.getElementById('value')).then((m) => {
+    console.error(m);
```

Comment:
same question

Reply:
can add it as a console log, it was just for kitchen sink app to print the meeting